tools/diagnostic-information

```diff
@@ -164,6 +164,14 @@ csv_export() {
     )
 }
 
+sqldump() {
+    verbose "sqldump..."
+    mysqldump box_utf8 >"${WD}/${TMPDIR}/dump.sql" || (
+        echo "Can't create SQL dump, is database running?" >&2
+        echo "Failed to create SQL dump" >"${WD}/${TMPDIR}/dump.sql"
```

Comment:
Better `>>` just in case we did get some SQL and then it got aborted for some reason.